bundles/org.smarthomej.binding.tr064/README.md

```diff
@@ -204,6 +212,16 @@ val tr064Actions = getActions("tr064","tr064:fritzbox:2a28aee1ee")
 val result = tr064Actions.phonebookLookup("49157712341234", 5)
 ```
 
+### Fritz!Box Backup
+
+The `fritzbox` things can create configuration backups of the Fritz!Box.
+The default configuration of the Fritz!Boxes require 2-factor-authentication for creating backups.
+To use the rule-action you have to disable this (but beware: depending on your configuration this might be a security issue).
```

Comment:
Do you have a link to instructions how it is working? I couldn't find something about it yet.

Reply:
done